index.html

```diff
@@ -68,17 +69,29 @@ <h4>Unit</h4>
           </div>
         </form>
       </div>
-      <div id="more-info-page" class="hidden"></div>
+      <div id="more-info-page" class="hidden">
+        <p>hello world</p>
+        <img src="http://www.7timer.info/bin/civil.php?lon=-117.8&lat=33.7&ac=0&lang=en&unit=british&output=internal&tzshift=0" alt="temp">
```

Comment:
"temp" isn't descriptive of what this image actually is. Does this mean temporary, or temperature? If it's the latter, then it should say temperature.